cms/migrations/0017_setup_home_page.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.2.10 on 2020-06-09 10:12
+
+from django.db import migrations
+
+DEFAULT_HOMEPAGE_PROPS = dict(title="Home Page", subhead="This is the home page")
```

Comment:
The default title should be "MIT Bootcamps", subhead should be "Learn Innovation and Entrepreneurship from MIT"

Reply:
Also, it should be `tagline`, not `subhead`.